Added documentation for keys used before 1.9

Flux3on opened the pull request:
kS1-kS15 were all deprecated in 1.9, but I thought it would be good to add them in anyway.

zmxhawrhbg commented:
don't forget the blending and player color keys!
`kA5` - col1 blending
`kS16` - BG pcol (0 if no pcol, 1/2 if pcol (number determines player), always exists)
`kS17` - Ground pcol
`kS18` - Line pcol
`kS19` - Obj pcol
`kS20` - col1 pcol
of course, the range between kS20 and kS29, from what i can tell, exists and likely relates to col2 and col3 (col4 was added in as one of the last things of 1.9, after the color format redo), but there will be 0 levels using it and i haven't experimented with it enough to give definitive keys

AlFasGD commented on docs/resources/client/level-components/inner-level-string.md:
```diff
@@ -49,6 +49,17 @@ The following keys were valid prior to 2.0 and are deprecated, since they are in
 
+***Pre-1.9 Keys***
+
+The following keys were valid prior to 1.9 and were deprecated as of 1.9, since they were included in the keys ks29-33. Each color, instead of using one key, used 3 keys to represent Red, Green, and Blue values.
```
When referring to keys and such, consider using the format `kS29`. Also, care for the capitalization.

In the Red, Green, Blue values, it's important to note that the aforementioned properties were respective to the keys, so change it to "... and Blue values, respectively."

Flux3on commented:
@AlFasGD I included what you wanted to be included, as well as adding the keys that @zmxhawrhbg mentioned in their comment.

AlFasGD commented on docs/resources/client/level-components/inner-level-string.md:
```diff
@@ -49,6 +49,25 @@ The following keys were valid prior to 2.0 and are deprecated, since they are in
 
+***Pre-1.9 Keys***
+
+The following keys were valid prior to 1.9 and were deprecated as of 1.9, since they were included in the keys kS29, kS30, kS31, kS32, and kS33. Each color, instead of using one key, used 3 keys to represent Red, Green, and Blue values, respectively. In 1.7 and 1.8, the game used a fourth key to determine if a color channel used a player color, and which player color it would use.
```
You still didn't change to `kS29`. Also, `kS29`-`kS33` should be just fine.

Flux3on replied:
I'm not sure what you mean by this. Should I remove "the keys?"

AlFasGD replied:
I meant to format using \`kS29\`, just like how I've been doing in these reviews. The other comment refers to simplifying the range into "\`kS29\`-\`kS33\`".

Flux3on replied:
Ohhh I see. Ok

AlFasGD commented on docs/resources/client/level-components/inner-level-string.md:
```diff
@@ -49,6 +49,25 @@ The following keys were valid prior to 2.0 and are deprecated, since they are in
+| kS7-kS9   | Line Color     | kS31       |
+| kS10-kS12 | Object Color   | kS32       |
+| kS13-kS15 | Obj-2 Color    | kS33       |
+| kS16      | BG p-col       | kS29       |
```
Change `p-col` to `Player Color`.

AlFasGD commented on docs/resources/client/level-components/inner-level-string.md:
```diff
@@ -49,6 +49,25 @@ The following keys were valid prior to 2.0 and are deprecated, since they are in
+| kS19      | Object p-col   | kS32       |
+| kS20      | Obj-2 p-col    | kS33       |
+
+*For keys kS16-kS20, if the key value is 0, neither player colors are used and the color that was selected in the color picker is used for that channel. If the value is 1, the color channel uses player color 1. If the value is 2, player color 2 is used.*
```
Convert to a value table instead of this long string.

Flux3on commented:
UPDATE: I changed the keys I added to use code markdown, changed "p-col" to "Player Color," and moved the possible values for `kS16`-`kS20` to their own table.

AlFasGD commented on docs/resources/client/level-components/inner-level-string.md:
```diff
@@ -51,22 +51,28 @@ The following keys were valid prior to 2.0 and are deprecated, since they are in
+| kS19      | Object Player Color | kS32       |
+| kS20      | Obj-2 Player Color  | kS33       |
+
+Keys `kS16`-`kS20` always used 1 of 3 values to determine if a player color was used, and if so, which one.
```
A better statement would be "Keys `kS16`-`kS20` used to determine the player color that was being used and their values were interpreted according to the following table:"

Flux3on commented:
I changed it to that verbatim

SMJSGaming commented:
I'll assume that these changes are ok now. @AlFasGD next time if everything is approved then merge the changes as well
